src/converters/degiroConverterV2.test.ts

```diff
@@ -5,13 +5,21 @@ import { GhostfolioExport } from "../models/ghostfolioExport";
 describe("degiroConverterV2", () => {
 
   beforeEach(() => {
-   // jest.spyOn(console, "log").mockImplementation(jest.fn());
+    // jest.spyOn(console, "log").mockImplementation(jest.fn());
+    // Mock YahooFinanceService - we don't want to make real requests and don't care about the result.
+    jest.spyOn(YahooFinanceService.prototype, "getSecurity").mockResolvedValue({
```

Comment:
Testing that comments are working using the "Add single comment" button.

Reply:
And now?